AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithUserPoolIntegrationTests/GraphQLWithUserPoolIntegrationTests.swift

```diff
@@ -42,18 +56,19 @@ class GraphQLWithUserPoolIntegrationTests: XCTestCase {
         Amplify.reset()
     }
 
-    let user1 = "user1@GraphQLWithUserPoolIntegrationTests.com"
-    let password = "Abc123@@!!"
+
     // This is a run once function to set up users then use console to verify and run rest of these tests.
     func testSetUpOnce() {
```

Comment:
Since running this by itself isn't sufficient to actually provision the test user, let's remove it altogether in favor of including CLI commands or console instructions in the test setup README.

Reply:
updating readme with instructions on setting up a user and removing disabled method